glossary.md

```diff
@@ -264,6 +264,9 @@ replaced by [Power Control Service (PCS)](#power-control-service-pcs).
 * For more information on CAPMC, see [Cray Advanced Platform Monitoring and Control](operations/power_management/Cray_Advanced_Platform_Monitoring_and_Control_CAPMC.md).
 * For more information on the CAPMC API, see [CAPMC API](api/capmc.md).
 
+CAPMC was deprecated in CSM 1.5 and was intended to be removed in CSM 1.6.
+[Power Control Service (PCS)](#power-control-service-pcs) is the replacement for CAPMC.
```

Comment:
This is just a suggestion. Only make a change if my suggestion or something else makes more sense to you.

The following matches closer to my understanding of the current state. I'm wondering if we told the customer that CAPMC was going to be removed. Either way it may not be important to talk about that, since we now plan to keep CAPMC in a deprecated state indefinitely.

[Power Control Service (PCS)](#power-control-service-pcs) is the replacement for CAPMC.
CAPMC is deprecated starting in CSM 1.5. CAPMC now passes all requests directly through to PCS.

Reply:
While we're not removing CAPMC, it's my understanding that we no longer intend to support it starting in 1.6.  I did update the wording slightly here.  Let me know if this looks ok (didn't add the bit about passing all requests through because it doesn't pass power capping requests through to PCS):

- CAPMC was deprecated in CSM 1.5 and no longer supported starting in CSM 1.6.
- [Power Control Service (PCS)](#power-control-service-pcs) is the replacement for CAPMC.

This matches what's currently in introduction/deprecated_features/CAPMC_Deprecation_Notice.md.  I'll ask Rod to review that file as well just to doubly make sure this is what's intended.

Reply:
It may simplify/shorten some of the text to not mention the planned removal of it in CSM 1.6. That doesn't seem relevant. It would probably suffice just to say something like "It has been EOL since CSM 1.3 and deprecated since CSM 1.5. In CSM 1.6, its use is strongly discouraged and it is no longer supported."

Reply:
Updated to state:

- CAPMC was deprecated in CSM 1.5 and no longer supported starting in CSM 1.6.